### PR TITLE
Gbl bugfix

### PR DESCRIPTION
**Pull request by RB-Development**
bugfix to gambling

### CODE REVIEW

File: src/ui_components/inventory.py [hunk 1]
```diff
@@ -13,7 +13,6 @@
 from utils.custom_mouse import mouse
 import os
 from ui_components import stash
-from ui_components.stash import gold_full
```
Comment (Ezro) [inline src/ui_components/inventory.py:16]: from ui_components.stash import set_gold_full

File: src/ui_components/inventory.py [hunk 2]
```diff
@@ -77,7 +76,7 @@ def stash_all_items(num_loot_columns: int, item_finder: ItemFinder, gamble = Fal
                     if stash.curr_stash["gold"] > 3:
                         #decide if gold pickup should be disabled or gambling is active
                         if Config().char["gamble_items"]:
-                            gold_full = True
```
Comment (Ezro) [inline src/ui_components/inventory.py:80]: set_gold_full(True)

**Review by Ezro — CHANGES_REQUESTED** [review]

**Comment by Ezro** [comment]
image

**Review by mgleed — APPROVED** [review]
Thank you. Made some small modifications per @Ezro 's recommendations and removed a redundant for loop I found.

**Pull request merged**